Sming/SmingCore/Network/TcpClient.h

```diff
@@ -54,6 +54,13 @@ class TcpClient : public TcpConnection
 	__forceinline bool isProcessing()  { return state == eTCS_Connected || state == eTCS_Connecting; }
 	__forceinline TcpClientState getConnectionState() { return state; }
 
+	inline void setReceiveCallback( TcpClientDataDelegate on_receive ) {
+		receive = on_receive;
+	}
+	inline void setCompleteCallback( TcpClientCompleteDelegate on_complete ) {
```

Comment:
Oops, you are right @slaff - I missed that somehow becaused it is not mentioned in the api documentation ...

So I will add a new method _setReceiveDelegate()_ only? Should both methods be `__forceinline` to improve consistency? Or should the new method go to _TcpClient.cpp_ as _setCompleteDelegate()_?